### PR TITLE
Website Redesign - Sending Data

### LINKED ISSUE #10912: Website Redesign - Sending Data
As a User,
I want to visit the ReportStream **Sending Data** Page,
So that I can become informed about the details around the process of sending data

Designs
- Figma Link: https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design

Content Updates
- [See Figma Link](https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design)

Page Title
- "CDC ReportStream - Sending Data" 

Analytics
- All links
- Page Load
- Accordion interactions

Storybook updates
- Accordion
- ProcessList

Verification Steps
- Signed off in Staging by Product / Design
- Signed off in Production by Product / Design / Engineering

Acceptance Criteria:
- [ ] Analytics / Metrics Required 
- [ ] Storybook update required
- [ ] Unit Tests
- [ ] End to End Tests
- [ ] Manual Testing Steps

Dependencies:
- Designs - https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design
- Platform API Endpoints - N/A
- Links - TBD

Additional Details:
- The deployment strategy will follow the general deployment strategy for the Website Redesign EPIC. We will deliver this page in isolation from the other pages. 
- Conversion to MDX will be determined by the Engineer owning this ticket
- Add Back To Top to frontmatter in MDX

### PR DESCRIPTION
**Pull request by jpandersen87**
Fixes #10912

This PR implements the /getting-started/sending-data.

### CODE REVIEW

**Review by etanb — APPROVED** [review]

File: frontend-react/src/content/getting-started/sending-data.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: Get started sending data
+subtitle: How to start simplifying public health reporting for your organization
```
Comment (penny-lischer) [inline frontend-react/src/content/getting-started/sending-data.mdx:3]: The font is incorrect. Should be 
color: var(--base, #71767A);

File: frontend-react/src/content/getting-started/sending-data.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+<CardHeader>
+### <A href="/assets/pdf/ReportStream_Onepager_20230621.pdf" target="_blank">ReportStream overview PDF <Icon name='FileDownload'/></A>
+</CardHeader>
+<CardBody>
+Download a quick summary of ReportStream that you can print and share.
+</CardBody>
+</Card>
+<Card gridLayout={{tablet: {col: 4}}}>
+<CardHeader>
+### [ReportStream API](/api-guide)
```
Comment (penny-lischer) [inline frontend-react/src/content/getting-started/sending-data.mdx:110]: should this be /api

**Comment by audreykwr** [comment]
- [ ] Receiving data in the side nav isn't working
- [ ] My own mistake - need to uncapitalize terms of service in 2 places under point 2 in the accordion. 
- [ ] "our API onboarding" link in point 4 of the accordion needs to go to the API guide page
- [ ] The SimpleHealth Kit quote doesn't match the Figma design: it should read “The documentation was great and the ReportStream folks were very helpful.”
- [ ] Kit in SimpleHealth Kit needs to be capitalized
- [ ] My mistake I believe - remove the quotation mark in the middle of the Dendi quote
- [ ] Remove the period at the end of the Dendi description
- [ ] Link under where we send data will go to the current version of the "where we're live", and that isn't working right now, but we'll need to change that url eventually when it is updated to "our network"
- [ ] API guide link in the bottom card is not working
- [ ] Release notes link is also not working

**Comment by yudonoodles** [comment]
On 08/25, @jpandersen87 and met to go over everything I am sharing below and walked through what needs to be updated together. The goal is to standardize spacing and have it be reflected in design as well as during implementation moving forward. 

**Spacing notes:**

[] Side panel -uses grid-col-3 and main content uses grid-col-9 that comes to a total of 12 columns. 
[] Spacing between side-navigation and main-content should be 40px
[] Add margin-top on <p> following H1 : 20px
[] Add margin-top on H2: 64px 
[] Add margin-top on <p> tag following H2: 20px
[] Add margin-top to accordion: 32px
[] Add margin-top to blockquote: 32px
[] Add padding-top, padding-bottom, padding-right, and padding-left to blockquote: 24px
[] Remove margin-top and margin-bottom 22.080px from text eg. “The documentation was there…”
[] Add margin-top to footer in blockquote: 32px 
[] Add margin-top to usa-process-list: 32px
[] Add margin-top to usa-card-group: 32px
[] Align Usa-card’s icon so that it is flush with bottom of the text, i.e. download icon with “ReportStream overview PDF”
[] Add margin-top to H3: 32px
[] Add margin-top to "back to top" link: 64px
* Noting here for future: notified Joseph that usa-card group appears to have additional spacing, but for now we determined it not so critical to resolve.

**Review by penny-lischer — APPROVED** [review]
lgtm

**Review by yudonoodles — APPROVED** [review]

**Pull request merged**